charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -31,6 +31,10 @@ spec:
               value: "{{- printf "%s-%s" .Release.Name "s3-data" | trunc 63 | trimSuffix "-" -}}"
             - name: REDIS_HOST
               value: "{{- printf "%s-%s" .Release.Name "redis" | trunc 63 | trimSuffix "-" -}}"
+            - name: CRR_METRICS_HOST
```

Comment:
Actually, we may want to change our code to simply use the environment variables Kubernetes already provides to look up service hosts/addresses and ports, instead of needing to always re-export it. Anyway, not a blocker for this PR.